_episodes/02-filedir.md

```diff
@@ -348,21 +356,18 @@ shell-lesson-data/
 
 Note that if a directory named `Desktop` does not exist in your current working directory,
 this command will return an error. Typically, a `Desktop` directory exists in your
-home directory, which we assume is the current working directory of your bash shell.
+home directory, which we assume is the current working directory of your shell.
```

Comment:
Emphasizing tutorial is for Bash is helpful, especially for those on Mac who will need to switch into Bash from Zsh.
```suggestion
home directory, which we assume is the current working directory of your bash shell.
```